pkg/eventmanager/eventmanager.go

```diff
@@ -29,12 +29,14 @@ const (
 	UPGRADE_EXTDEPCHECK_DELAY_DESC = "Cluster upgrade to version %s is experiencing a delay as an external dependency of the upgrade is currently unavailable. The upgrade will continue to retry. This is an informational notification and no action is required by you."
 	// UPGRADE_SCALE_DELAY_DESC describes the upgrade scaling delayed
 	UPGRADE_SCALE_DELAY_DESC = "Cluster upgrade to version %s is experiencing a delay attempting to scale up an additional worker node. The upgrade will continue to retry. This is an informational notification and no action is required by you."
+	// UPGRADE_SCALE_DELAY_SKIP_DESC describes the upgrade scaling skipped after delay
+	UPGRADE_SCALE_DELAY_SKIP_DESC = "Cluster upgrade to version %s has experienced an unforeseen issue during capacity reservation efforts. The upgrade will continue without extra compute. This is an informational notification and no action is required by you."
```

Comment:
Is this truly an unforeseen/unexpected issue? What types of scenarios is it possible that we could run into this issue? Saying that something is "unforeseen" makes it appear that we were lazy or not thorough enough. I'd just like to be as explicit as possible.

Reply:
Hi @wgordon17 was trying to implement the dynamic message by reading the failure status from the machine. But turns out that the MachineHealthCheck currently deployed in our cluster will remove the machine as well as the machine status.
Based on our discussion, the reason could be AWS service quota issue, temporary node connectivity issue to the master or to the external dependency services, like quay. 

Would you like to review the new message? Thanks!

Reply:
@spurtell Could you help take a look here?